strings.lisp

```diff
@@ -143,6 +143,10 @@ From Emacs Lisp."
           (unless (= (prog1 i (incf i)) ult)
             (write-string sep stream))))))
 
+;; FIXME: FUN might be a funcallable object, thus no typing for
+;; it. Better/portable way to allow funcallables?
+(-> mapconcat (t sequence string-designator &key (:stream (or null stream)))
+    string)
```

Comment:
This signature is too simple and the actual signature is probably too complicated to be useful, so this should be omitted. STREAM can be anything format can handle as a stream; also mapconcat may not return a string if it is writing to a stream.

Also AFAIK funcallable objects are a subtype of function.

Reply:
Okay, the signature is removed :)